lib/msal-browser/src/telemetry/BrowserPerformanceClient.ts

```diff
@@ -19,7 +19,7 @@ export class BrowserPerformanceClient extends PerformanceClient implements IPerf
         this.guidGenerator = new GuidGenerator(this.browserCrypto);
     }
     
-    startPerformanceMeasurement(measureName: string, correlationId: string): IPerformanceMeasurement {
+    startPerformanceMeasuremeant(measureName: string, correlationId: string): IPerformanceMeasurement {
```

Comment:
Sorry to bother. What was the cause of the bug looking back? This change looks a bit odd since if I'm not mistaken measurement is spelled measurement and not measuremeant.

Reply:
@ralphtheninja Yes the spelling will be corrected in a future release.